editor/code/src/client.ts

```diff
@@ -239,6 +244,14 @@ export function activate(context: ExtensionContext): void {
     }
   };
 
+  const setFailedStatuBar = (emsg: string) => {
+    lspStatusItem.text = "$(circle-slash) coq-lsp (client failed to start)";
```

Comment:
```suggestion
    lspStatusItem.text = "$(circle-slash) coq-lsp (error)";
```
The tooltip should be enough. The recommendation is not to take up to much space.

Reply:
I see, but I find `error` too generic, what error? As this is the "exceptional" case, IMHO it is fine to be a bit longer, but I hear your concern.

Actually the message is wrong, it is the server that failed.

Reply:
Made it a bit shorter.